Result.podspec

```diff
@@ -9,7 +9,6 @@ Pod::Spec.new do |s|
   s.source       = { :git => 'https://github.com/antitypical/Result.git', :tag => s.version }
   s.source_files  = 'Result/*.swift'
   s.requires_arc = true
-  s.pod_target_xcconfig = { 'SWIFT_VERSION' => '2.3' }
```

Comment:
This should probably explicitly specify 3.0 (or whatever the correct value is) for the `SWIFT_VERSION`.